public/index.html

```diff
@@ -7,36 +7,234 @@
-                <img src="../assets/social icons/Vector (3).svg" alt="">
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/1.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/2.png" alt="">
+        <h2>Multi-Post
+            Stories</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/3.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
```

Comment:
- [ ] Kindly, add button interaction like `hover` `active` `enabled`. This will make you project more interacctive and user friendly 📱😄